data/json/items/comestibles.json

```diff
@@ -2422,12 +2422,12 @@
         "symbol" : "%",
         "quench" : -1,
         "healthy" : 1,
-        "nutrition" : 40,
+        "nutrition" : 24,
```

Comment:
It's more like dried and rehydrated fish are completely broken in the opposite way.